security/file_integrity_operator/fio-overview.adoc

```diff
@@ -0,0 +1,21 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="fio-overview"]
+= File Integrity Operator overview
+include::_attributes/common-attributes.adoc[]
+:context: fio-overview
+
+The File Integrity Operator that continually runs file integrity checks on the cluster nodes. It deploys a DaemonSet that initializes and runs privileged link:https://aide.github.io/[Advanced Intrusion Detection Environment] (AIDE) containers on each node, providing a log of files that have been modified since the initial run of the DaemonSet pods.

```

Comment:
```suggestion
The File Integrity Operator continually runs file integrity checks on the cluster nodes. It deploys a DaemonSet that initializes and runs privileged link:https://aide.github.io/[Advanced Intrusion Detection Environment] (AIDE) containers on each node, providing a log of files that have been modified since the initial run of the DaemonSet pods.

```